### DISCUSSION TITLE
How to download

### DISCUSSION THREAD

**Discussion by Val6363** [community/community#42873] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How do I upload the coding into my betterdiscord plugins? I can't download the 'files', it just takes me to the code

**Comment by Val6363** [comment]
How do I download this https://github.com/TheViralClovers/discordEmojiGrabber
I don't have linux and can't get it. I have a mac

**Reply by SaileshBK** [reply]
There is a drop down option called Download ZIP try that one and extract the file after.

**Reply by Alpha2Chris14** [reply]
There should be a place to download it has a zip file after which you can extract

**Comment by radar-blip** [comment]
If you look at the README.md for that repository, it has instructions for Windows, Linux, and Mac.

**Comment by arjunj132** [comment]
The repo has instructions for mac [here](https://github.com/TheViralClovers/discordEmojiGrabber#mac). Also, you can click `Code` and click `Download ZIP`.
